buckets/buckets.go

```diff
@@ -49,12 +50,22 @@ type Bucket struct {
 	Key       string   `json:"_id"`
 	Name      string   `json:"name"`
 	Path      string   `json:"path"`
+	EncKey    string   `json:"key,omitempty"`
```

Comment:
A bucket's encryption key is stored on the model (base64 encoded 32 bytes AES, 32 bytes HMAC)